### ISSUE TITLE
Syncthing: Failed to load/generate certificate: save cert: open /config/cert.pem: permission denied

### ISSUE THREAD

**Issue by luandro** [digidem/edt-offline#29]
Error:
```
[start] 2023/03/16 12:33:10 INFO: syncthing v1.23.2 "Fermium Flea" (go1.19.6 linux-arm64) root@buildkitsandbox 2023-03-07 19:03:19 UTC [noupgrade]
[start] 2023/03/16 12:33:10 INFO: Generating ECDSA key and certificate for syncthing...
[start] 2023/03/16 12:33:10 WARNING: Failed to load/generate certificate: save cert: open /config/cert.pem: permission denied
```

Probably related to Balena Starter Interface which is also accessing the same volumes as syncthing and filebrowser. Checking user ownership and modifying how that works should be the fix.

Worst case remove shared volumes from starter interface.

**Comment by luandro** [comment]
This was caused because the `PUID` and  `PGID` Balena vars weren't being automatically added to the container. A new release with the new `balena.yml` should fix this.

**Comment by luandro** [comment]
Solved by #28
